scripts/evtx_dump_json.py

```diff
@@ -4,82 +4,94 @@
 #
 #   Purpose: User can dump evtx data into JSON format to either the command line or a JSON file in new line delimited format/JSON array.
 #   Details: The JSON object is created with only the EventRecordID from the System section of the evtx XML and all of the information within the EventData section.
-#
-#   Requires:
-#     - xmltodict >= 0.12.0
-import os
-import json
-
-import xmltodict
 
 import Evtx.Evtx as evtx
+import Evtx.Views as e_views
+
+# Added packages
+import os
+import xmltodict
+import json
 
 
 def main():
     import argparse
-
-    parser = argparse.ArgumentParser(description="Dump a binary EVTX file into XML.")
-    parser.add_argument("evtx", type=str, action="store", help="Path to the Windows EVTX event log file")
-    parser.add_argument("-o", "--output", type=str, action="store", help="Path of output JSON file")
+    parser = argparse.ArgumentParser(
+        description="Dump a binary EVTX file into XML.")
+    parser.add_argument("evtx", type=str,action='store',
+                        help="Path to the Windows EVTX event log file")
+    parser.add_argument("-o","--output",type=str, action='store',
+                        help="Path of output JSON file")
     args = parser.parse_args()
 
     with evtx.Evtx(args.evtx) as log:
 
         # Instantiate the final json object
-        final_json = []
+        final_json=[]
```

Comment:
there are a lot of inconsistent formatting changes in this PR. would you please revert the ones that don't influence the program logic?

Reply:
made changes to revert back, sorry about that!